### ISSUE TITLE
[Optional Request] A better logo may be

### ISSUE THREAD

**Issue by sudovijay** [SaraVieira/awesome-talks#51]
This one just doesn't have that feel, i'm not saying its bad as i'm no designer, it just doesn't fit with the site theme 🙂

**Comment by SaraVieira** [comment]
You are right, I made that on a rush , I'll tweet this out and see if anyone is willing to help 🙌

**Comment by sudovijay** [comment]
[just a reminder] you need to tweet girl lol or make a new one, this one just not right. 🙂
